modelon/impact/client/sal/request.py

```diff
@@ -38,26 +38,41 @@ def __init__(
         self.headers = headers
 
     def execute(self, check_return: bool = True) -> Any:
+        cookies = self.context.session.cookies
+        access_token = cookies.get("access_token")
+        session_cookies = {"access_token": access_token}
```

Comment:
This is only needed and should only be done for "localhost" otherwise we should just keep the "session.cookies"

Reply:
Good point! Fixed it! :)